compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/fbt/fbt-to-string.js

```diff
@@ -0,0 +1,15 @@
+import fbt from "fbt";
+
+function Component(props) {
+  const element = (
+    <fbt desc={"Dialog to show to user"}>
+      Hello <fbt:param name="user name">{props.name}</fbt:param>
+    </fbt>
+  );
+  return element.toString();
```

Comment:
failed previously because element has the inferred type BuiltInJsx, which didn't have a shape definition